packages/manager/apps/dedicated/client/app/account/user/users/sso/details/user-users-sso-details.html

```diff
@@ -0,0 +1,64 @@
+<oui-header heading="{{:: 'user_users_sso_detail_title' | translate }}">
+</oui-header>
+<div>
+    <a
+        class="d-inline-block col-md-12 mb-12"
+        data-ng-href="{{'#/useraccount/users' }}"
+        title="{{ 'user_users_sso_detail_back' | translate }}"
+        style="margin-bottom: 1rem;"
+    >
+        <span class="fa fa-chevron-left mr-2" aria-hidden="true"> </span>
+        <span data-translate="user_users_sso_detail_back"></span>
+    </a>
+
+    <div
+        class="col-md-8 mb-8"
+        data-ng-if="identityProvider != null"
+        style="margin-bottom: 3rem;"
+    >
+        <oui-tile
+            data-heading="{{:: 'user_users_sso_detail_idp_title' | translate }}"
+        >
+            <oui-tile-definition
+                data-term="{{:: 'user_users_sso_detail_idp_sso_service_url' | translate }}"
+                data-description="{{:: identityProvider.ssoServiceUrl }}"
+            >
+            </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'user_users_sso_detail_idp_group_attribute_name' | translate }}"
+                data-description="{{:: identityProvider.groupAttributeName}}"
+            >
+            </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'user_users_sso_detail_idp_creation' | translate }}"
+                data-description="{{:: identityProvider.creation | date }}"
+            >
+            </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'user_users_sso_detail_idp_last_update' | translate }}"
+                data-description="{{:: identityProvider.lastUpdate | date }}"
+            >
+            </oui-tile-definition>
+            <oui-tile-definition
+                data-term="{{:: 'user_users_sso_detail_idp_signing_certificates' | translate }}"
+            >
+            </oui-tile-definition>
+            <ul
+                data-ng-repeat="idpSigningCertificate in identityProvider.idpSigningCertificates"
+            >
+                <li>
+                    <strong
```

Comment:
**nitpick:** Prefer using `translate` directive (`data-translate`) and `ng-bind` (instead of `{{`)

Reply:
Done